File: src/main/java/org/compiere/order/MShippingTransactionLine.java
package org.compiere.order;

import kotliquery.Row;

public class MShippingTransactionLine extends X_M_ShippingTransactionLine {

    /**
     *
     */
    private static final long serialVersionUID = -8148869412130244360L;

    public MShippingTransactionLine(
            int M_ShippingTransactionLine_ID) {
        super(M_ShippingTransactionLine_ID);
    }

    public MShippingTransactionLine(Row row) {
        super(row);
    }

    public void setADClientID(int AD_Client_ID) {
        super.setADClientID(AD_Client_ID);
    }
}

File: src/main/kotlin/org/compiere/order/MShippingTransactionLine.kt
package org.compiere.order

import kotliquery.Row

class MShippingTransactionLine : X_M_ShippingTransactionLine {

    constructor(
        M_ShippingTransactionLine_ID: Int
    ) : super(M_ShippingTransactionLine_ID) {
    }

    constructor(row: Row) : super(row) {}

    companion object {
        private const val serialVersionUID = -8148869412130244360L
    }
}
